documentation/site/pages/provers/quick-start.mdx

```diff
@@ -126,42 +126,42 @@ cargo install --locked --git https://github.com/boundless-xyz/boundless boundles
 ### Deposit Collateral
 
 :::warning
-*$ZKC* is only on Ethereum mainnet. 
+*$ZKC* is only on Ethereum mainnet.
 
-Therefore, to use *$ZKC* as proving collateral, provers have to first [bridge *$ZKC*](/zkc/collateral) from Ethereum mainnet to Base mainnet. 
+Therefore, to use *$ZKC* as proving collateral, provers have to first [bridge *$ZKC*](/zkc/collateral) from Ethereum mainnet to Base mainnet.
 :::
 
 With the environment variables set, you can now deposit *$ZKC* tokens as collateral to your account balance:
 
 ```bash [Terminal]
-boundless account deposit-collateral 10
+boundless account deposit-collateral 50
 ```
 
-### Start Broker
+### Start Prover
 
-You can now start broker (which runs both bento + broker i.e. the full proving stack!):
+You can now start the prover (which runs both bento + broker i.e. the full proving stack!):
 
 ```bash [Terminal]
-just broker
+just prover
```

Comment:
Noting was delaying making this docs change until the release was ready, so as long as we release soon after this is merged this is fine